packages/network-controller/src/NetworkController.ts

```diff
@@ -55,6 +75,7 @@ export type NetworkState = {
   isCustomNetwork: boolean;
   providerConfig: ProviderConfig;
   properties: NetworkProperties;
+  networkConfigurations: Record<string, NetworkConfiguration>;
```

Comment:
Do we want to add this to the JSDoc above?

Reply:
done [here](https://github.com/MetaMask/core/pull/1064/commits/ee8cfdcdd7743c9a3318ff52c845ea82101e5dd7)